packages/fxa-settings/src/pages/Signin/SigninBounced/index.tsx

```diff
@@ -27,7 +26,6 @@ export const viewName = 'signin-bounced';
 
 const SigninBounced = ({
   email,
-  emailLookupComplete,
```

Comment:
Never noticed this before... Seems like some sort of race condition workaround. Glad it's gone! Is there something we do now that means the race condition is no longer a problem? (This quesiton is loaded with assupmtions. Feel free to just dismiss it if it's out of line.)

Reply:
I left a related comment here (2nd paragraph): https://github.com/mozilla/fxa/pull/15621#discussion_r1280989537

My change here was a bit of scope creep when I was refactoring some of App. I noticed we were basically duplicating logic that we already had in `currentAccount()` - we were doing this in a `useEffect` before which I believe caused the need for a useState, but it doesn't look like we need it. Since Signin routes are still WIP, it felt low risk to change.